### PR TITLE
Feat: Navigation

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added Navigation bar with primary color

### CODE REVIEW

**Review by KanekarSnehal — APPROVED** [review]

File: components/Navigation/style.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  background: var(--primary-color);
+  width: 100%;
+  height: 3.125rem;
+  align-items: center;
+  display: flex;
+  padding: 0px 0.625rem;
+  justify-content: space-between;
+  color: #ffff;
+  display: flex;
+  flex-direction: row !important;
```
Comment (nerdyspook) [inline components/Navigation/style.css:16]: Here you have used display 2 times, you can avoid these kinds of mistakes by writing properties related to each other in groups. Like: 

```
width: 100%;
height: 3.125rem;

display: flex;
flex-direction: row;
justify-content: space-between
```

File: components/Navigation/style.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.header_menu{
+  display: flex;
+  align-items: center;
+  flex-direction: row !important;
```
Comment (devananth) [inline components/Navigation/style.css:4]: Try to avoid using `!important`

**Review by nerdyspook — CHANGES_REQUESTED** [review]

File: components/Navigation/style.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+.header_component {
+  background: var(--primary-color);
+  width: 100%;
+  height: 3.125rem;
+  align-items: center;
+  display: flex;
+  padding: 0px 0.625rem;
+  justify-content: space-between;
+  color: #ffff;
```
Comment (devananth) [inline components/Navigation/style.css]: you use `css variables` here too.

**Review by devananth — CHANGES_REQUESTED** [review]
Suggested few changes , have a look. 

Good work !

**Review by nerdyspook — APPROVED** [review]

**Pull request merged**